src/lib/select/select.ts

```diff
@@ -131,6 +132,9 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
   /** Subscription to tab events while overlay is focused. */
   private _tabSubscription: Subscription;
 
+  /** Subscription to global scrolled events while the select is open. */
+  private _scrolledSubscription: Subscription;
```

Comment:
Nit: Mind changing this to `_scrollSubscription`? For some reason, I keep tripping over it as is because it sounds like past tense.